src/client/Microsoft.Identity.Client/TokenCacheNotificationArgs.cs

```diff
@@ -55,5 +57,15 @@ public sealed partial class TokenCacheNotificationArgs
         /// See https://aka.ms/msal-net-app-cache-serialization for details.
         /// </remarks>
         public bool IsApplicationCache { get; }
+
+        /// <summary>
+        /// Indicates whether MSAL's current in-memory token cache contains any tokens. The in-memory cache is        
+        /// kept in sync with the persisted version through OnBefore / OnAfter callbacks. 
+        /// </summary>
+        /// <remarks>
+        /// If this flag is false in the OnAfterAccessAsync notification, the token cache can be deleted.        
+        /// MSAL takes into consideration access tokens expiration when computing this flag, but not refresh token expiration, which is not known to MSAL.7
```

Comment:
Proposing to remove the 7 :)

```suggestion
        /// MSAL takes into consideration access tokens expiration when computing this flag, but not refresh token expiration, which is not known to MSAL.
```